include/seqan3/io/sequence/sequence_file_format_fasta.hpp

```diff
@@ -212,8 +213,10 @@ class sequence_file_format_fasta
         static_assert(detail::decays_to_ignore_v<qual_type> || detail::decays_to_ignore_v<seq_qual_type>,
                       "Either the qualities field, or the seq_qual field need to be set to std::ignore.");
 
-        auto stream_view = ranges::iterator_range{std::istreambuf_iterator<char>{stream},
-                                                  std::istreambuf_iterator<char>{}};
+        auto stream_view = view::subrange<decltype(std::istreambuf_iterator<char>{stream}),
```

Comment:
template argument deduction never works for type aliases :sob: 

They want to fix it, but it won't be before C++23, I would wager.

Reply:
Oh I see, well that makes sense

Reply:
If this starts to annoy us seriously in the future, we can think about inheriting instead of aliasing, but I think we would then also need to redefine the deduction guides.